src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import CityDisplay from './CityDisplay';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayCityData: false,
+      cityData: [],
+      userSearch: '',
+      errMsg: '',
+      errRender: false,
```

Comment:
Good work on the lab so far! 

- Feedback here is to review state here as well as in `CityDisplay`, we have the same state in both components. When you can, grab a TA so they can help you decide where state is stored 👍